packages/web/src/scss/components/UNSTABLE_EmptyState/index.html

```diff
@@ -13,14 +13,14 @@ <h2 class="docs-Heading">Default</h2>
                   stroke-dasharray="4 4"></path>
           </svg>
           <div class="docs-Placeholder__text">
-            <div class="typography-heading-xsmall-text">Placeholder</div>
-            <p class="typography-body-medium-text-regular">Replace me with your own component</p></div>
+            <div class="typography-headline-xsmall-regular">Placeholder</div>
```

Comment:
I see body medium bold in Figma:

![image](https://github.com/user-attachments/assets/72ecaf64-5a12-42ca-a395-b58c06c78920)